### ISSUE TITLE
OUT OF SCOPE: Component - Stack List - issue with NVDA

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#27]
Although NVDA is not the departmental standard, there is an issue with this component when using it:

• Items are not read as they slide in. 
• Should have aria-live attribute.

With NVDA New information is not announced to screen reader user. Screen readers have to navigate backwards in order to hear the new information presented on the screen.
JAWS seems to be fine.

**Comment by ftourangeau** [comment]
This component has multiple accessibility issues and was deactivated from our Adapt instance
